plugin/plugin.xml

```diff
@@ -1,271 +1,531 @@
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.buildship.core.files.gradlebuildscript"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wildwebdeveloper.json"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wildwebdeveloper.yaml"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wildwebdeveloper.ts"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wildwebdeveloper.tsx"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wildwebdeveloper.js"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wst.dtd.core.dtdsource"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wst.xml.core.xmisource"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.eclipse.wst.xml.core.xmlsource"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-    <contentTypeMapping
-      contentType="org.python.pydev.pythonfile"
-      id="io.snyk.languageserver">
-    </contentTypeMapping>
-  </extension>
-  <extension
+    <extension
+        name="%extension.name"
+        point="org.eclipse.ui.views">
+        <category
+            name="%category.name"
+            id="io.snyk.eclipse.plugin">
+        </category>
+        <view
+            id="io.snyk.eclipse.plugin.views.SnykView"
+            name="%view.name"
+            icon="icons/patch.png"
+            class="io.snyk.eclipse.plugin.views.SnykView"
+            category="io.snyk.eclipse.plugin"
+            inject="true">
+        </view>
+        <view
+            id="io.snyk.eclipse.plugin.views.snyktoolview"
+            name="%toolView.name"
+            icon="icons/patch.png"
+            class="io.snyk.eclipse.plugin.views.snyktoolview.SnykToolView"
+            category="io.snyk.eclipse.plugin"
+            inject="true">
+        </view>
+    </extension>
+    <extension point="org.eclipse.ui.perspectiveExtensions">
+        <perspectiveExtension
+            targetID="org.eclipse.jdt.ui.JavaPerspective">
+            <view
+                id="io.snyk.eclipse.plugin.views.SnykView"
+                relative="org.eclipse.ui.views.ProblemView"
+                relationship="right"
+                ratio="0.5">
+            </view>
+            <view
+                id="io.snyk.eclipse.plugin.views.snyktoolview"
+                relative="org.eclipse.ui.views.ProblemView"
+                relationship="right"
+                ratio="0.5">
+            </view>
```

Comment:
I'd not automatically add it to the perspective right now, so it does not appear in the Preview automatically without function.